packages/misc/src/lib/misc.ts

```diff
@@ -667,25 +665,15 @@ export const defaultMintClaimCallback: MintCallback<
   network: string = 'cayenne'
 ): Promise<string> => {
   try {
-    let relayUrl: string = '';
-
-    switch (network) {
-      case LitNetwork.Cayenne:
-        relayUrl = RELAY_URL_CAYENNE + '/auth/claim';
-        break;
-      case LitNetwork.Habanero:
-        relayUrl = RELAY_URL_HABANERO + 'auth/claim';
-        break;
-      case LitNetwork.Manzano:
-        relayUrl = RELAY_URL_MANZANO + 'auth/claim';
-        break;
-      case LitNetwork.DatilDev:
-        relayUrl = RELAY_URL_DATIL_DEV + '/auth/claim';
-        break;
-        break;
-      case LitNetwork.DatilTest:
-        relayUrl = RELAY_URL_DATIL_TEST + '/auth/claim';
-        break;
+    const AUTH_CLAIM_PATH = '/auth/claim';
+
+    const relayUrl: string =
+      RELAYER_URL_BY_NETWORK[network as LIT_NETWORK_VALUES] + AUTH_CLAIM_PATH;
+
+    if (!params.relayUrl && !relayUrl) {
```

Comment:
This will always be `false` because of `AUTH_CLAIM_PATH`

![Screenshot from 2024-07-11 15-52-13](https://github.com/LIT-Protocol/js-sdk/assets/10857348/b5690f51-ffaf-4d4a-946f-493d859422c1)

However, I think that the best error we can return here is that the URL is broken. Letting this pass and failing at the request, which will also catch the cases where user passes an invalid `params.relayUrl`

Reply:
Nice catch! You mean always return `true` because `relayUrl` is always a valid string right?